test/network/network_test.cpp

```diff
@@ -85,7 +86,8 @@ class NetworkTests : public TerrierTest {
       server_ =
           std::make_unique<TerrierServer>(common::ManagedPointer<ProtocolInterpreter::Provider>(&protocol_provider_),
                                           common::ManagedPointer(handle_factory_.get()),
-                                          common::ManagedPointer(&thread_registry_), port_, connection_thread_count_);
+                                          common::ManagedPointer(&thread_registry_), port_, connection_thread_count_,
+                                          true /* Use unix domain socket */, socket_directory_);
```

Comment:
Does using the unix domain socket here force it to be used on all the tests? If yes, I would recommend making a new file to test the domain socket behavior individually and rename this file to correspond to internet socket tests.

Reply:
No. While the Unix domain socket is created and left open for all network tests, it only actually gets tested by the one test I've created.

Reply:
Gotcha, could you clarify the difference in setup in network_test.cpp:196 that causes the Unix domain socket to be used here in a comment.